call_opt_test.go

```diff
@@ -0,0 +1,92 @@
+package jsonrpc2_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/sourcegraph/jsonrpc2"
+)
+
+func TestPickID(t *testing.T) {
```

Comment:
I created this `call_opt_test.go` file and moved `TestPickID` over from `jsonrpc2_test.go`. I did not make any changes to `TestPickID` though.